caffe2/utils/threadpool/pthreadpool-cpp.cc

```diff
@@ -59,6 +93,11 @@ size_t getDefaultNumThreads();
 PThreadPool* pthreadpool() {
   static std::unique_ptr<PThreadPool> threadpool =
       std::make_unique<PThreadPool>(getDefaultNumThreads());
+#ifndef WIN32
+  // Register pthread_atfork handlers to prevent segfaults in worker processes
+  // after fork.
+  pthread_atfork(before_fork, after_fork, nullptr);
```

Comment:
This would register new callback every time this function is called, please used std::once